### PR TITLE
Update for IPAudioMode and IPVideoMode - New-CsTeamsMeetingPolicy

### PR DESCRIPTION
**Pull request by doshidev**
Update New-CsTeamsMeetingPolicyfor new meeting policy setting IPAudioMode and IPVideoMode

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -150,6 +150,34 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -IPAudioMode
+Determines whether audio can be turned on in meetings and group calls. Set this to ENABLEDOUTGOINGINCOMING to allow outgoing and incoming audio in the meeting. Set this to DISABLED to prohibit outgoing and incoming audio in the meeting.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:159]:
```suggestion
Aliases:

```

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 2]
```diff
@@ -150,6 +150,34 @@ Accept pipeline input: False
...
+Accept wildcard characters: False
+```
+
+### -IPVideoMode
+Determines whether video can be turned on in meetings and group calls. Set this to ENABLEDOUTGOINGINCOMING to allow outgoing and incoming video in the meeting. Set this to DISABLED to prohibit outgoing and incoming video in the meeting.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
```
Comment (dariomws) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:173]:
```suggestion
Aliases:

```

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]
I found only one small detail.

Thank you again @doshidev!

**Review by get-itips — APPROVED** [review]
I am ok with it, maybe we can mention that you can't set ENABLEDOUTGOINGINCOMING on one of the parameters and DISABLED on the other.
"Invalid value combination IPVideoMode: EnabledOutgoingIncoming and IPAudioMode: Disabled."

**Pull request merged**